### PR TITLE
Add Toast Notifications List styling and test page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
You can view the new test page at: http://rawgit.com/jeff-phillips-18/patternfly/toast-dist/dist/tests/toast-notifications-list.html

File: tests/pages/toast-notifications-list.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </label>
+    <label class="radio-inline">
+      <input type="radio" name="optradio" value="warning">Warning
+    </label>
+    <label class="radio-inline">
+      <input type="radio" name="optradio"value="danger">Danger
+    </label>
+    <div class="checkbox">
+      <label><input type="checkbox" value="">Show Close</label>
+    </div>  </form>
```
Comment (dgutride) [inline tests/pages/toast-notifications-list.html]: Closing form tag ended up on the wrong line (probably not worth fixing).

**Comment by dgutride** [comment]
@jeff-phillips-18  - should the close button work when included on rawgit?

**Comment by jeff-phillips-18** [comment]
@dgutride No. I did not implement close. That exercise is left to the reader 😉

**Comment by jeff-phillips-18** [comment]
Fixed closing form tag ( @dgutride issue). Might as well fix it now... 🍰

**Comment by jeff-phillips-18** [comment]
@dgutride OK, I added closing of the toast notifications.

**Comment by dgutride** [comment]
Looks good to me

File: tests/pages/toast-notifications-list.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          '      <span class="pficon pficon-close' + '">' +
+          '      </span>' +
+          '    </button>';
+      }
+      toastHtml +=
+        '  </div>' +
+        '</div>';
+      $('.toast-notifications-list-pf').append(toastHtml);
+      $('#' + toastId).find('#closeButton').click(function () {
+        console.log(toastId);
```
Comment (dlabrecq) [inline tests/pages/toast-notifications-list.html]: Can the log message be removed? I see a bunch of "toast-notification-4" messages in my console that don't mean anything to me.

**Comment by jeff-phillips-18** [comment]
Removed extraneous log message ( @dlabrecq issue )

**Comment by priley86** [comment]
I give this a 💯  . LGTM 👍

**Comment by dlabrecq** [comment]
LGTM

**Pull request merged**